clients/src/main/java/org/apache/kafka/clients/consumer/KafkaConsumer.java

```diff
@@ -1319,6 +1319,15 @@ public OffsetAndMetadata committed(TopicPartition partition) {
         }
     }
 
+    public Map<TopicPartition, OffsetAndMetadata> committed(Set<TopicPartition> partitions) {
```

Comment:
Thanks for catching this. This method is not required anymore as it's part of rejected alternatives 1 and 2. I'll remove it.